services/qrexec2.md

```diff
@@ -127,7 +127,7 @@ Whenever a RPC request for service named "XYZ" is received, the first line
 in `/etc/qubes-rpc/policy/XYZ` that matches the actual `srcvm`/`destvm` is
 consulted to determine whether to allow RPC, what user account the program
 should run in target VM under, and what VM to redirect the execution to. If
-the policy file does not exits, user is prompted to create one *manually*;
+the policy file does not exist, user is prompted to create one *manually*;
```

Comment:
also this statement is no longer true since https://github.com/QubesOS/qubes-core-admin-linux/pull/24